source/deploying-with-kubernetes/kubernetes-conf.rst

```diff
@@ -0,0 +1,327 @@
+**Pods**
+
+    .. code-block:: bash
+
+        $ kubectl get pods -n wazuh
+        NAME                              READY     STATUS    RESTARTS   AGE
+        wazuh-elasticsearch-0             1/1       Running   0          15m
+        wazuh-kibana-f4d9c7944-httsd      1/1       Running   0          14m
+        wazuh-logstash-777b7cd47b-7cxfq   1/1       Running   0          13m
+        wazuh-manager-master-0            1/1       Running   0          12m
+        wazuh-manager-worker-0-0          1/1       Running   0          11m
+        wazuh-manager-worker-1-0          1/1       Running   0          11m
+        wazuh-nginx-748fb8494f-xwwhw      1/1       Running   0          14m
+
+**Accesing Kibana**
+
+    In case you created domain names for the services, you should be able to access Kibana using the proposed domain name: https://wazuh.your-domain.com.
+
+    Also, you can access using the External-IP (from the VPC): https://192.168.10.5.elb.amazonaws.com
+
+    .. code-block:: bash
+
+        $ kubectl get services -o wide -n wazuh
+        NAME                  TYPE           CLUSTER-IP       EXTERNAL-IP                                                    PORT(S)                          AGE       SELECTOR
+        wazuh-nginx           LoadBalancer   xxx.xx.xxx.xxx   internal-xxx-yyy.us-east-1.elb.amazonaws.com                   80:31831/TCP,443:30974/TCP       15m       app=wazuh-nginx
+
+.. note::
+    `AWS route 53 <https://aws.amazon.com/route53/?nc1=h_ls>`_ can be used to create a DNS that points to the load balancer and make it accessible through that DNS.
+
+Agents
+------
+
+Monitoring hosts
+^^^^^^^^^^^^^^^^
+
+Wazuh agents are designed to monitor hosts. To start using them:
+
+1. :doc:`Install the agent <../installation-guide/installing-wazuh-agent/index>`.
+
+
+2. Now, register the agent using the :doc:`registration service <../user-manual/registering/use-registration-service>`.
+
+
+3. Modify the file ``/var/ossec/etc/ossec.conf``, changing the "transport protocol" to *TCP* and changing the ``MANAGER_IP`` for the external IP of the service pointing to port 1514 or for the DNS provided by *AWS Route 53* if you are using it.
+
+
+4. Using the `authd <https://documentation.wazuh.com/current/user-manual/reference/daemons/ossec-authd.html?highlight=authd>`_ daemon with option *-m* specifying the external IP of the Wazuh service that takes to the port 1515 or its DNS if using *AWS Route 53*.
+
+
+.. note::
+    If using EKS remember to configure the security groups to manage the access.
+
+Monitoring containers
+^^^^^^^^^^^^^^^^^^^^^
+
+The objective here is to monitor the container from the node that has raised it.
+
+In this case, we have 2 options:
+
+    - Running the agent in the container: containers are sealed and designed to run a single process. It is not practicable solution.
+    
+    - Install the agent on the host: This is the option that we recommend since the agent was originally designed for this purpose.
```

Comment:
The goal here is to monitor the container from the node that has built it.

The next two options are valid:

     - Install the agent on the host, this is the recommended option since the agent was originally designed for this purpose.
     - Running the agent in the container: containers are sealed and designed to run a single process. It's not the best solution.